src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -857,6 +857,12 @@ def _decode_nonce(backend, nonce):
     OCSPExtensionOID.NONCE: _decode_nonce,
 }
 
+_OCSP_SINGLERESP_EXTENSION_HANDLERS_NO_SCT = _REVOKED_EXTENSION_HANDLERS.copy()
```

Comment:
Does the RFC say that anything valid for a revoked extension can be a single response extenion, or does it _happen_ to be that case?

Reply:
It explicitly says it: https://tools.ietf.org/html/rfc6960#section-4.4.5

Reply:
Please add a comment